java-frontend/src/main/java/org/sonar/java/se/xproc/MethodYield.java

```diff
@@ -189,13 +189,16 @@ public boolean equals(Object obj) {
 
   private Set<SymbolicValue> getSymbolicValues(List<Integer> parameterIndices) {
     ImmutableSet.Builder<SymbolicValue> parameterSVs = ImmutableSet.builder();
-    for (Integer parameterIndex : parameterIndices) {
+    parameterIndices.stream()
+      // Ignore parameters for varargs
```

Comment:
that comment is wrong. We should only ignore the last parameters, starting from the variadic index.